MarvelCharsApp/MarvelCharsApp/Models/Entities/MarvelAPIModel.swift

```diff
@@ -8,18 +8,18 @@
 // This file was generated from JSON Schema using quicktype, do not modify it directly.
 // To parse the JSON, add this file to your project and do:
 //
-//   let marvel = try? newJSONDecoder().decode(Marvel.self, from: jsonData)
+//   let category = try? newJSONDecoder().decode(Category.self, from: jsonData)
 
 import Foundation
 
-// MARK: - MarvelElement
-struct MarvelElement: Codable {
-    let heroes, villains, antiHeroes, aliens: [Alien]
-    let humans: [Alien]
+// MARK: - CategoryElement
+struct CategoryElement: Codable {
+    let name, key: String
+    let characterList: [CharacterList]
 }
 
-// MARK: - Alien
-struct Alien: Codable {
+// MARK: - CharacterList
+struct CharacterList: Codable {
```

Comment:
shouldn't this struct be named "Character"?

Reply:
Yes, it can be "Character". "CharacterList" was the name for the new property in the json who we define in the class on last week. I can change it locally refactoring the code and push again.

Reply:
sure, please do It, struct name and variable names doesn't require to be the same, and single name fits better in this case because of its properties

Reply:
The changes was committed Jose